docs/_articles/en/codebuilder/cb-setup.md

```diff
@@ -3,42 +3,88 @@ title: Set Up Code Builder
 lang: en
 ---
 
-## Install the Code Builder Managed Package
+Enable Code Builder to provide the permissions needed to install the Code Builder managed package in a Salesforce supported org.
 
-Salesforce Code Builder is a second-generation (2GP) managed package that a user with requisite permissions can install in any supported Salesforce org edition.
+### Required Editions
```

Comment:
@sbudhirajadoc - Do you have any concerns with duplicating the edition info here?  Should we just point to the H&T page with this info instead?

Reply:
We only have edition information here, @smaddox-sf . This is true for both Einstein for Developers and Code Builder.

Reply:
to be clear, dev ops center has their entire doc on H&T and not on developer.salesforce.com and we are the opposite.